tests/integration/security/pass_through_filter_chain_test.go

```diff
@@ -106,11 +135,25 @@ func TestPassThroughFilterChain(t *testing.T) {
 				{
 					target: b,
 					port:   8085,
+					schema: protocol.HTTP,
 					want:   true,
 				},
 				{
 					target: b,
 					port:   8086,
+					schema: protocol.HTTP,
+					want:   false,
+				},
+				{
+					target: b,
+					port:   8087,
+					schema: protocol.TCP,
+					want:   true,
```

Comment:
The comment says only 8085 is allowed. is the comment wrong or is the code wrong?

Reply:
Sorry, I missed to fix the comment. The authz policy checks port and only ports 8085 and 8087 are allowed.

Reply:
Done. Changed the comment.

Reply:
Ah I missed that. Thanks